packages/sku/src/services/vite/createConfig.ts

```diff
@@ -0,0 +1,78 @@
+import { builtinModules } from 'node:module';
+import type { InlineConfig } from 'vite';
+
+import react from '@vitejs/plugin-react';
```

Comment:
Can we use `@vitejs/plugin-react-swc` instead? Appreciate I might be throwing a spanner in the works if you've been testing with the regular react plugin this whole time, but we already have a dependency on SWC, so we might as well lean into that and get the perf benefits from using it for react too.